contrib/cf-remote/cf_remote/ssh.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
```

Comment:
I don't think a hash-bang is needed in this file.

Reply:
Thanks